docs/LLAMA_CPP_SETUP_AND_TESTING.md

```
@@ -0,0 +1,899 @@
+    uuid-dev \
+    pkg-config
+
+# Verify installations
+cmake --version      # Should be >= 3.20
+g++ --version        # Should be >= 9
+pkg-config --version
+```
+
+### Step 1.3: Install llama.cpp
+
+**Option A: Package Manager (Recommended)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)